src/components/Common/TitleAndMediaList.js

```diff
@@ -59,7 +59,7 @@ const TitleAndMediaList = ({
       </MediaItems>
       {CTALink && CTAText ? (
         <Padding top={3}>
-          <StyledLink external href={CTALink} title={CTAText}>
+          <StyledLink to={CTALink} title={CTAText}>
```

Comment:
Keep in mind this is a Common component and used throughout the site not just for the medium post. I think we need to add props to this component to be able to switch between `external` and `to` props when required.

Reply:
Thank you for this 🙌